src/server/views/summary.html

```diff
@@ -17,18 +26,16 @@ <h1 class="govuk-heading-l">
         {{ summaryDetail(detail) }}
       {% endfor %}
 
-      {% if not result.error  %}
-        <form method="post" autocomplete="off" novalidate>
-          <input type="hidden" name="crumb" value="{{crumb}}"/>
+      <form method="post" autocomplete="off" novalidate>
+        <input type="hidden" name="crumb" value="{{crumb}}"/>
```

Comment:
We only really do this for crumb, but one to replace the XHTML `/>` → `>` another day

```suggestion
        <input type="hidden" name="crumb" value="{{crumb}}">
```